### PR TITLE
cryptodetails page setup

### PR DESCRIPTION
**Pull request by seahro**
after finding out what my "category" error was ... I had to change the chartjs version.  the way I setup the dataset was compatible with chartjs 2.9.4 not chartjs 4.0.0 or chartjs 3.6. (I had 4 installed, then tried 3.6...2.9.4 solved my issue).

### CODE REVIEW

File: src/components/LineChart.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { Line } from 'react-chartjs-2';
+import { Col, Row, Typography } from 'antd';
+
+const { Title } = Typography;
+
+const LineChart = ({ coinHistory, currentPrice, coinName }) => {
+    const coinPrice = [];
+    const coinTimestamp = [];
+
+    for(let i = 0; i < coinHistory?.data?.history?.length; i += 1) {
```
Comment (stephenrawson) [inline src/components/LineChart.jsx:11]: Nice, there are many ways of iterating through a list.  For loops are just the tip of the iceberg.

File: src/components/LineChart.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    const data = {
+        labels: coinTimestamp,
+        datasets: [
+            {
+                label: 'Price in USD',
+                data: coinPrice,
+                fill: false,
+                backgroundColor: '#0071bd',
+                borderColor: '#0071bd'
```
Comment (stephenrawson) [inline src/components/LineChart.jsx:24]: Confused here? Your dataset probably doesn't need to contain colors. Maybe move to the component that displays the dataset

**Review by stephenrawson — APPROVED** [review]

File: src/components/LineChart.jsx (hunk 2 above)
Comment (seahro) [reply src/components/LineChart.jsx:24]: This was where my whole issue came up (this was a 2.9.4 format).  the 'data: coinPrice' and the "labels: coinTimestamp" are the only data items that i brought into the chart.   The colors and fill, rest are the arbitrary values that I selected. 
Once I am done with this test, I am going to reinstall the newer chartJS back and try some new charts for the next round of coding...

**Comment by seahro** [comment]
Thanks!

**Pull request merged**